content/docs/01-introduction/05-palette-modes.md

```diff
@@ -0,0 +1,88 @@
+
+
+# What is App Mode?
+
+App Mode is a unique experience enabled by Palette that aims to remove Kubernetes infrastructure overhead as much as possible. In app mode, you, as the consumer, instead focus on creating and managing App Profiles. App profiles are declarative templates that you can use to define all the required services, containers, and databases that make up an application. Once an app profile is defined, you can deploy the application to any Palette Virtual Cluster by specifying the respective app profile.
+
+App mode comes with an out-of-the-box cluster group managed by us here at Spectro Cloud called beehive.  This cluster group, which under the cover is a collection of Kubernetes clusters, is configured to support Palette Virtual Clusters. As a consumer, you can deploy a new virtual cluster to the beehive cluster group and get started with a Kubernetes cluster in minutes. 
+
+The ability to get started with a Kubernetes cluster in minutes is what makes app mode a powerful development tool. You can use the virtual clusters in an ephemeral manner, such as testing, ad-hoc development, or any other scenario where you would want a short-lived Kubernetes environment up and running in minutes.  
+
+Alternatively, you could also use App Mode to offer your own Palette-managed host clusters as a PaaS experience to downstream consumers. This concept is easier explained through an example. Assume you are a system administrator, and you want to expose Kubernetes to various in-house development teams. You could deploy several Kubernetes clusters to various platforms and create a " development " cluster group. You also ensured every cluster is enabled for Palette Virtual Cluster by selecting the option before deployment. You can now direct your organization members to use app mode and create Palette Virtual Clusters as needed, or you can create virtual clusters ahead of time for them. The organization members or downstream consumers can now focus on creating app profiles and deploying their applications. You have essentially, at this point, enabled a Kubernetes PaaS experience for your organization.
+
+This makes App mode unique from Cluster Mode, the ability to leverage an out-of-the-box host cluster group managed by us here at Spectro Cloud, or you can leverage Palette-managed Kubernetes clusters you already have deployed configured to your organization’s standards and expose them as Kubernetes Paas experience. As the consumer of app mode, you simply focus on deploying your application to a Kubernetes cluster by specifying the app profile. The overhead of managing infrastructure has essentially been removed for you, thus freeing up your time to focus on what matters the most, developing an application that solves business problems.   
```

Comment:
This paragraph repeats and may not be needed. If you keep part or all of it, 'app mode' needs to be capitalized.

Reply:
I removed the first two sentences to reduce the repetition.